en/docs/learn/api-security/threat-protection/bot-detection.md

```diff
@@ -1,8 +1,10 @@
 # Bot Detection
 
-After a Publisher publishes APIs in the API Developer Portal, hackers can invoke the APIs without an access token by scanning the open ports of a system. Therefore, WSO2 API Manager has a bot detection mechanism in place to prevent such attacks by identifying who tried to enter and invoke resources without proper authorization. WSO2 API Manager's bot detection mechanism traces and logs details of such unauthorized API calls and sends notifications in this regard via emails. Thereby this helps Publishers to protect their data from bot attackers and improve the security of their data.  
+There is a possibility that hackers may invoke services without any proper authentication by using tools such as port scanning.  Therefore, WSO2 API Manager(WSO2 API-M) provides a bot detection mechanism in place.
```

Comment:
port scanning is a method AFAIK. Port scanner is a tool for that. Shall we change `tools` in to `mechanisms` or 'methods`?

Reply:
Do port scanning tools invoke the services without authentication or do port scanning tools scan the open ports of the system and detect an unadvertised service and bots try to access it without proper authentication?

Reply:
Yeah agree with @dushaniw. Let's reword it to convey that

Reply:
port scanning helps attachers to identify services exposed through open ports and can then attempt to exploit potential vulnerabilities in any services they find.

Port scanners ~= Port Scanning tools 

Port scanners such as Nmap, NetCat are used for port scanning. So they are basically port scanning tools.

Reply:
I would reword the phase as 

There is a possibility that hackers may exploit services exposed in the gateway by identifying such services using tools such as port scanners.

Reply:
fixed.